frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -492,7 +492,7 @@ export function StatusTag({ survey }: { survey: Survey }): JSX.Element {
     } as Record<ProgressStatus, LemonTagType>
     const status = getSurveyStatus(survey)
     return (
-        <LemonTag type={statusColors[status]} style={{ fontWeight: 600 }} data-attr="status">
```

Comment:
Wonder how this crept in past the eslint rule? It was only added 3 months ago

Reply:
It looks like the rule only applies to props on HTML elements, and not React components 🤔